lib/ansible/utils/yaml.py

```diff
@@ -0,0 +1,32 @@
+# (c) 2020 Matt Martz <matt@sivel.net>
```

Comment:
Can you move this abstraction into module_utils? That will allow it to be used in the [legacy_collection_loader](https://github.com/ansible/ansible/tree/devel/test/lib/ansible_test/_data/legacy_collection_loader) (which also needs to be updated), and will also avoid having to make an exception for this import when running modules/module_utils unit tests.

If there are any modules that happen to use libyaml, this will also be useful for them as well.

Reply:
Yeah, I can do that.  I've been meaning to get back to this PR.